modules/setup/setup.js

```diff
@@ -119,29 +153,38 @@ class Setup {
                         return;
                     }
                     progressbar.update(progress);
-                    }, 1000);
+                    }, FUNCTION_CALL_INTERVAL);
+                    
                     docker.modem.followProgress(dockerPullStream, (_dockerModemError, _dockerModemOutput) => {
                         clearInterval(progress);
                         progressbar.update(100);
-                        console.log(Helper.output('\nTezos nodes have been setup successfully on system....'));
+                        Logger.info('\nTezos nodes have been setup successfully on system....');
                         process.exit();
                     });
                 }
         });                     
     }
 
     runContainer(){
-        this.startNodesProgressBar();
+        Setup.startNodesProgressBar();
```

Comment:
Why all these methods are static methods? 
You need to call these methods with this.startNodesProgressBar(). 
If we have to make a method static to use in the same class, we're doing something wrong.